lib/perl/Genome/Model/Tools/SpeedSeq/Align.pm

```diff
@@ -0,0 +1,78 @@
+package Genome::Model::Tools::SpeedSeq::Align;
+
+use strict;
+use warnings;
+
+use Genome;
+
+class Genome::Model::Tools::SpeedSeq::Align {
+    is => 'Genome::Model::Tools::SpeedSeq::AlignBase',
+    has_input => [
+        fastq => {
+            is => 'Text',
+            doc => 'paired-end fastq file. if -p flag is used then expected to be an interleaved paired-end fastq file, and fastq2 may be omitted. (can be gzipped)',
+            example_values => ['in1.fq'],
+        },
+        fastq2 => {
+            is => 'Text',
+            doc => 'paired-end fastq file. (can be gzipped)',
+            is_optional => 1,
+            example_values => ['in2.fq'],
+        }
+    ],
+    has_param => [
+        read_group_header => {
+            is => 'Text',
+            doc => 'read group BAM header line',
+            example_values => ['@RG\tID:id\tSM:samplename\tLB:lib'],
+        },
+    ],
+};
+
+
+sub execute {
+    my $self = shift;
+
+    # OPTIONS aka params
+    my $options = $self->_resolve_options_string('-R "'. $self->read_group_header .'"');
+
+    # INPUTS
+    my @input_files;
+    my $inputs_string = $self->reference_fasta .' '. $self->fastq;
+    push @input_files, $self->reference_fasta;
+    push @input_files, $self->fastq;
+    if ($self->config_file) {
+        push @input_files, $self->config_file;
+    }
+    if ($self->paired) {
+        if ($self->fastq2) {
+            die('Expected single interleaved paired-end FASTQ!');
+        }
+        $options .= ' -p';
```

Comment:
It's a bit strange that one option is passed into `_resolve_options_string`, most are resolved inside there, and this one is tacked on here.

One possibility is that this command could override `_resolve_options_string` to add its own options.  Another might be a system like `GMT::Picard` uses to derive them.